tensorflow-core/tensorflow-core-api/src/test/java/org/tensorflow/ConcreteFunctionTest.java

```diff
@@ -0,0 +1,122 @@
+package org.tensorflow;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+import org.tensorflow.op.Ops;
+import org.tensorflow.op.core.Init;
+import org.tensorflow.op.core.Placeholder;
+import org.tensorflow.op.math.Add;
+import org.tensorflow.op.math.Sub;
+import org.tensorflow.types.TFloat32;
+
+public class ConcreteFunctionTest {
+
+  private static Signature plusFive(Ops tf) {
+    Placeholder<TFloat32> input = tf.placeholder(TFloat32.DTYPE);
+    Add<TFloat32> output = tf.math.add(input, tf.constant(5.0f));
+    Init init = tf.init();  // for native resource management tests
```

Comment:
What does this init trigger?

Reply:
Nothing, like the comment says, it is only used to test out resource management as in `closingFunctionCreatedFromGraphOnlyReleaseResourcesItOwns`